python_ta/cfg/visitor.py

```diff
@@ -145,3 +145,11 @@ def _visit_continue_or_break(self, node: Union[astroid.Break, astroid.Continue])
         unreachable_block = self._current_cfg.create_block()
         self._current_block = unreachable_block
 
+    def visit_return(self, node: astroid.Return) -> None:
+        if type(self._current_cfg.start.statements[0]).__name__ != astroid.FunctionDef.__name__:
```

Comment:
Overall I'd say it's better not to use this approach, but instead treat a `FunctionDef` as a new *control boundary* and add an exit point for `'Return'` to jump to the end of the function.